quickstart/docker/image/run-router.sh

```diff
@@ -30,10 +30,11 @@ if [[ "${_ZITI_ROUTER_NAME}" != "" ]]; then
   export ZITI_ROUTER_NAME="${_ZITI_ROUTER_NAME}"
   echo "ZITI_ROUTER_NAME set to: ${ZITI_ROUTER_NAME}"
 fi
-_UNIQUE_NAME="${ZITI_HOME}/${ZITI_ROUTER_NAME}-${HOSTNAME}.init"
-if [ ! -f "${_UNIQUE_NAME}" ]; then
-  echo "system has not been initialized. initializing..."
-  "${ZITI_BIN_DIR-}/ziti" edge login ${ZITI_CTRL_EDGE_ADVERTISED_ADDRESS}:${ZITI_CTRL_EDGE_ADVERTISED_PORT} -u $ZITI_USER -p $ZITI_PWD -y
+
+"${ZITI_BIN_DIR-}/ziti" edge login ${ZITI_CTRL_EDGE_ADVERTISED_ADDRESS}:${ZITI_CTRL_EDGE_ADVERTISED_PORT} -u $ZITI_USER -p $ZITI_PWD -y
```

Comment:
shouldn't the login be skipped if the router config exists? seems like there's no need to login if you don't need to make a config to me?

Reply:
The login is used later to check if the router was enrolled, if not, it enrolls.

Currently the two (check for config and enrollment) were done in one go but I figured there may be a situation where a container goes down before enrollment or enrollment fails for some reason. Separating them out will check for a config but also check for enrollment even if the config is there.

Reply:
Right. I don't think that's the flow we want though. If the file exists, i think we expect it's been created in the overlay like the old logic.